features/dashboard.feature

```diff
@@ -0,0 +1,45 @@
+Feature: Admin dashboard
+
```

Comment:
Could you put the users story for the dash board feature here ????

Reply:
Yes sir!